### ISSUE TITLE
Missing psutil module dependency in requirements.txt

### ISSUE THREAD

**Issue by skorotkov** [confluentinc/ducktape#419]
@confluentinc/devprod-apac-n-frameworks-eng is tagged for visibility

**Describe the bug**

The https://github.com/confluentinc/ducktape/pull/416  PR merge adds dependency on the 'psutil' module.

But the requirements.txt file wasn't updated accordingly.

So the following error occurs if ducktape run in the prod venv configured via the **requirements.txt**:
```
Traceback (most recent call last):
  File "/usr/local/bin/ducktape", line 5, in <module>
    from ducktape.command_line.main import main
  File "/usr/local/lib/python3.9/dist-packages/ducktape/command_line/main.py", line 30, in <module>
    from ducktape.tests.runner import TestRunner
  File "/usr/local/lib/python3.9/dist-packages/ducktape/tests/runner.py", line 29, in <module>
    from ducktape.tests.runner_client import run_client
  File "/usr/local/lib/python3.9/dist-packages/ducktape/tests/runner_client.py", line 24, in <module>
    import psutil
ModuleNotFoundError: No module named 'psutil'
```

Note that the **requirements-test.txt** does contain 'psutil' module, so unit tests work fine.

**To Reproduce**

Run the following in the source tree:

```
python3 -m venv ./ducktape-venv
source ./ducktape-venv/bin/activate
pip install -r requirements.txt
pip install -e .
ducktape
```

**Expected behavior**
It's expected that all needed dependencies (including the psutil module) are installed via the requirements.txt.

**Comment by nagpavan-chilakam** [comment]
PR has been approved and merged to master and also 0.12.x branch

**Comment by skorotkov** [comment]
Sorry for delay. Yes sure the issue may be closed now.

Thanks much!
